src/Administration/Resources/app/administration/src/app/component/media/sw-image-slider/index.js

```diff
@@ -132,7 +132,7 @@ Component.register('sw-image-slider', {
             const link = this.isImageObject(image) ? image.src : image;
 
             try {
-                URL(link);
+                new URL(link);
```

Comment:
Hi @s-diez , you may just disable the linter in this case: `// eslint-disable-next-line no-new`
We're using this method of validating URLs in the core already.